ganttproject/src/net/sourceforge/ganttproject/gui/ResourceAssignmentsTableModel.java

```diff
@@ -24,13 +24,12 @@
 import net.sourceforge.ganttproject.task.ResourceAssignmentMutator;
 import net.sourceforge.ganttproject.task.Task;
 
-import javax.swing.table.AbstractTableModel;
 import java.util.*;
 
 /**
  * @author Oleg Kushnikov
  */
-public class ResourceAssignmentsTableModel extends AbstractTableModel {
+public class ResourceAssignmentsTableModel<T> extends TableModelExt {
```

Comment:
Did you mean to write concrete type, not type variable? `ResourceAssignmentsTableModel extends TableModelExt<ResourceAssignment>` ?